src/containers/profile-page/components/desktop/ProfilePage.tsx

```diff
@@ -399,7 +399,6 @@ const ProfilePage = ({
           ) : (
             <Lineup
               {...getLineupProps(artistTracks)}
-              count={profile.track_count}
```

Comment:
The count will be determined by the TrackProvider based on the number of lineup tracks (this is already done for mobile)